plugins/Google/test.py

```diff
@@ -34,34 +34,7 @@
 class GoogleTestCase(ChannelPluginTestCase):
     plugins = ('Google', 'Config')
     if network:
-        def testSearch(self):
-            self.assertNotError('google foo')
-            self.assertRegexp('google dupa', r'dupa')
-            # Unicode check
-            self.assertNotError('google ae')
-
-        def testUrlDecode(self):
-            self.assertRegexp(
-                    'google site:http://www.urbandictionary.com carajo land',
-                    '\x02carajo land - Urban Dictionary\x02: '
-                    r'https?://www.urbandictionary.com/define.php\?term=carajo%20land')
-
-        def testLucky(self):
-            self.assertResponse('lucky Hacker News',
-                    'https://news.ycombinator.com/')
-
-        def testSearchFormat(self):
-            self.assertRegexp('google foo', '<https?://.*>')
-            self.assertNotError('config reply.format.url %s')
-            self.assertRegexp('google foo', 'https?://.*')
-            self.assertNotRegexp('google foo', '<https?://.*>')
-
-        def testSearchOneToOne(self):
-            self.assertRegexp('google dupa', ';')
-            self.assertNotError('config plugins.Google.oneToOne True')
-            self.assertNotRegexp('google dupa', ';')
-
         def testTranslate(self):
-            self.assertRegexp('translate en es hello world', 'Hola mundo')
```

Comment:
Why remove the translation check?

Reply:
Google now returns "Hola Mundo" instead of "Hola mundo" which means the test was actually failing. The output is subject to change, so I'm not sure we should hardcode it.

Reply:
`(?i)` should make it case-insensitive

Reply:
Actually no, when I run the test locally, it shows "Hola World". While on my production bot (hosted elsewhere) it shows "Hola Mundo"?

Reply:
:shrug: let's allow both options

Reply:
I don't get it. The website shows multiple possible outputs too, but "Hola World" seems plainly wrong. Yet 
I can't reproduce that output with wget ...

![image](https://github.com/user-attachments/assets/3e979c97-ac14-4bde-aa07-27f002f926e6)

If we can't be sure what the API will return I don't think we should hardcode any expectation at all.

Reply:
but at least it checks we're not returning complete nonsense. And it doesn't run on the CI anyway

Reply:
Ok I'll just leave it in even though it doesn't pass for me